app/controllers/cloud_volume_snapshot_controller.rb

```diff
@@ -67,6 +71,42 @@ def show_list
     process_show_list
   end
```

Comment:
the `button` method is really ugly

we need a simple structure where a simple method is called for each button id